### PR TITLE
Sentence splicing removed. Filemask.

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: weblate/templates/trans/alert/nomaskmatches.html [hunk 1]
```diff
@@ -4,6 +4,6 @@
 <p>{% trans "There are no translations in this component." %}</p>
```
Comment (comradekingu) [inline weblate/templates/trans/alert/nomaskmatches.html:4]:
```suggestion
<p>{% trans "This component has no translations." %}</p>
```
Yet? Don't know how this is used.

File: weblate/templates/trans/alert/nomaskmatches.html [hunk 2]
```diff
@@ -4,6 +4,6 @@
...
 
 <ul>
-  <li>{% blocktrans with filemask=component.filemask %}Check whether file mask <code>{{ filemask }}</code> matches files in your repository.{% endblocktrans %}</li>
-  <li>{% trans "You can also start new translations directly in Weblate, it will create files based on the file mask." %}</li>
+  <li>{% blocktrans with filemask=component.filemask %}Check whether the filemask <code>{{ filemask }}</code> matches files in your repository.{% endblocktrans %}</li>
```
Comment (comradekingu) [inline weblate/templates/trans/alert/nomaskmatches.html:7]:
```suggestion
  <li>{% blocktrans with filemask=component.filemask %}Check whether the filemask <code>{{ filemask }}</code> matches files in your VCS repository.{% endblocktrans %}</li>
```

Comment (nijel) [reply weblate/templates/trans/alert/nomaskmatches.html:7]: It has shown in the past that many people don't understand what VCS stands for.

Comment (comradekingu) [reply weblate/templates/trans/alert/nomaskmatches.html:7]: Didn't know where to put the dash, but I think it is correct now.

File: weblate/templates/trans/alert/nomaskmatches.html [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {% load i18n %}
-<p>{% trans "This component set up is incomplete as it lacks translations." %}</p>
+<p>{% trans "This component set up is incomplete as it lacks any translations." %}</p>
```
Comment (comradekingu) [inline weblate/templates/trans/alert/nomaskmatches.html:2]:
```suggestion
<p>{% trans "This component set up is incomplete, as it lacks any translations." %}</p>
```

File: weblate/templates/trans/alert/nomaskmatches.html [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
 
 <ul>
   {% if analysis.can_add %}
-    <li>{% trans "Create new translations directly in Weblate, it will create files based on the file mask." %}</li>
+    <li>{% trans "Starting new translations directly in Weblate is possible, and creates files based on the filemask." %}</li>
   {% endif %}
-  <li>{% blocktrans with filemask=component.filemask %}Check whether file mask <code>{{ filemask }}</code> matches files in your repository.{% endblocktrans %}</li>
+  <li>{% blocktrans with filemask=component.filemask %}Check whether filemask <code>{{ filemask }}</code> matches files in your version-control system repository.{% endblocktrans %}</li>
```
Comment (comradekingu) [inline weblate/templates/trans/alert/nomaskmatches.html:8]:
```suggestion
  <li>{% blocktrans with filemask=component.filemask %}Check whether the filemask <code>{{ filemask }}</code> matches files in your version-control system repository.{% endblocktrans %}</li>
```

File: weblate/templates/trans/alert/nomaskmatches.html [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
-<p>{% trans "This component set up is incomplete as it lacks translations." %}</p>
+<p>{% trans "This component set up is incomplete, as it lacks any translations." %}</p>
 
 <ul>
   {% if analysis.can_add %}
-    <li>{% trans "Create new translations directly in Weblate, it will create files based on the file mask." %}</li>
+    <li>{% trans "Starting new translations directly in Weblate is possible, and creates files based on the filemask." %}</li>
   {% endif %}
-  <li>{% blocktrans with filemask=component.filemask %}Check whether file mask <code>{{ filemask }}</code> matches files in your repository.{% endblocktrans %}</li>
+  <li>{% blocktrans with filemask=component.filemask %}Check whether the filemask <code>{{ filemask }}</code> matches files in your version-control system repository.{% endblocktrans %}</li>
```
Comment (nijel) [inline weblate/templates/trans/alert/nomaskmatches.html:8]: Let's be consistent with the rest of the documentation:

```suggestion
  <li>{% blocktrans with filemask=component.filemask %}Check whether the filemask <code>{{ filemask }}</code> matches files in your version control system repository.{% endblocktrans %}</li>
```

**Pull request merged**